aiobotocore/__init__.py

```diff
@@ -1,4 +1,4 @@
 from .session import get_session, AioSession
 
-__version__ = '0.1.1'
+__version__ = '0.1.2'
```

Comment:
Minor suggestion, lets tag not released version as `0.1.2a0` or something like that, so we can reason better about versions in dev and in wild